DataFormats/BeamSpot/src/classes_def.xml

```diff
@@ -5,6 +5,11 @@
   </class>
   <class name="edm::Wrapper<reco::BeamSpot>"/>
 
-  <class name="BeamSpotPOD" persistent="false"/>
-  <class name="edm::Wrapper<BeamSpotPOD>" persistent="false"/>
+  <class name="BeamSpotPOD" ClassVersion="3">
+   <version ClassVersion="3" checksum="280341519"/>
+  </class>
+  <class name="edm::Wrapper<BeamSpotPOD>"/>
+
+  <class name="BeamSpotHost" persistent="true"/>
+  <class name="edm::Wrapper<BeamSpotHost>" persistent="true"/>
```

Comment:
```suggestion
  <class name="BeamSpotHost"/>
  <class name="edm::Wrapper<BeamSpotHost>"/>
```
(the other should work, but this is more along what we do elsewhere)

Doesn't the `BeamSpotHost` need the IO read rule (in either XML or C++ code)?